aiida/restapi/common/utils.py

```diff
@@ -442,18 +446,19 @@ def build_response(self, status=200, headers=None, data=None):
         response.status_code = status
 
         if headers is not None:
-            for k, v in headers.items():
-                response.headers[k] = v
+            for key, val in headers.items():
+                response.headers[key] = val
```

Comment:
for next time, possibly use `respone.headers.update(headers)` as a shorthand

Reply:
Here response.header is not a type of dict so we can not directly update it with another dictionary.